.github/workflows/ci-nix.yml

```diff
@@ -465,7 +468,7 @@ jobs:
           done
 
   pkgs:
-    if: github.repository == 'fedimint/fedimint' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/releases') || github.ref_type == 'tag')
+    if: github.repository == 'fedimint/fedimint' && (github.event_name == 'merge_group' || startsWith(github.ref, 'refs/heads/releases') || github.ref_type == 'tag')
```

Comment:
Do we have a real need for `merge_group`? I think we can get by with just the release branch and tags, and arguably just the tags

Reply:
Good catch, I modified everything a bit while thinking about it.